products/appengine/terraform.yaml

```diff
@@ -64,7 +64,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         test_env_vars:
           org_id: :ORG_ID
   Service: !ruby/object:Overrides::Terraform::ResourceOverride
-    exclude: true
+    id_format: "{{service_id}}"
+    import_format: ["apps/{{project}}/services/{{service_id}}"]
+    mutex: "apps/{{project}}/services/{{service_id}}"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_create: templates/terraform/custom_create/noop_on_create_appengine_version.go.erb
```

Comment:
Not sure whether I need to add create_encoder in this case. So there are two things:
1. Having updateMask in the create_url, I think this can be handled by just adding the constant like:
```
    create_url: 'apps/{{project}}/services/{{service_id}}?{{migrateTraffic}}&updateMask=split'
````
2. Identifying if we need to add shardBy also in the updateMask (I am looking into it)